rust-toolchain.toml

```diff
@@ -0,0 +1,3 @@
+[toolchain]
+channel = "1.78.0"
```

Comment:
As far as I know, we currently support Rust v1.75, so I think the version here should be v1.75.0